secretsmanager/include/secret_data.h

```diff
@@ -20,10 +20,12 @@ namespace concord::secretsmanager {
 
 // This struct contains the secret parameters needed to initialise the SecretsManagerEnc
 struct SecretData {
-  std::string password;  // Secret used for key generation
-  std::string digest;    // Digest algorithm used for key generaetion
-  std::string algo;      // Encryption algorithm used by SecretsManagerEnc
-  uint32_t key_length;   // The size of the key in bytes
+  std::string key;      // symmetric key
```

Comment:
Nit: Maybe mention that `key` and `iv` are hex strings in comments as opposed to something like Base64.